packages/highcharts-theme/css/salt-highcharts-theme.css

```
@@ -0,0 +1,133 @@
+/* Use of CSS Variables is not supported in Highcharts v10.2.0 (available in v11.0.0 onwards - https://code.highcharts.com/11.0.0/css/highcharts.css)*/
+/* .highcharts-theme-salt {
+  --highcharts-background-color: none;
+
+  --highcharts-color-0: var(--salt-palette-categorical-3);
+  --highcharts-color-1: var(--salt-palette-categorical-10);
+  --highcharts-color-2: var(--salt-palette-categorical-12);
+  --highcharts-color-3: var(--salt-palette-categorical-1);
+  --highcharts-color-4: var(--salt-palette-categorical-16);
+  --highcharts-color-5: var(--salt-palette-categorical-7);
+  --highcharts-color-6: var(--salt-palette-categorical-9);
+  --highcharts-color-7: var(--salt-palette-categorical-2);
+  --highcharts-color-8: var(--salt-palette-categorical-20);
+  --highcharts-color-9: var(--salt-palette-categorical-6);
+} */
+
+.highcharts-theme-salt .highcharts-color-0 {
+  fill: var(--salt-palette-categorical-3);
+  stroke: var(--salt-palette-categorical-3);
+}
+
+.highcharts-theme-salt .highcharts-color-1 {
+  fill: var(--salt-palette-categorical-10);
+  stroke: var(--salt-palette-categorical-10);
+}
+
+.highcharts-theme-salt .highcharts-color-2 {
+  fill: var(--salt-palette-categorical-12);
+  stroke: var(--salt-palette-categorical-12);
+}
+
+.highcharts-theme-salt .highcharts-color-3 {
+  fill: var(--salt-palette-categorical-1);
+  stroke: var(--salt-palette-categorical-1);
+}
+
+.highcharts-theme-salt .highcharts-color-4 {
+  fill: var(--salt-palette-categorical-16);
+  stroke: var(--salt-palette-categorical-16);
+}
+
+.highcharts-theme-salt .highcharts-color-5 {
+  fill: var(--salt-palette-categorical-7);
+  stroke: var(--salt-palette-categorical-7);
+}
+
+.highcharts-theme-salt .highcharts-color-6 {
+  fill: var(--salt-palette-categorical-9);
+  stroke: var(--salt-palette-categorical-9);
+}
+
+.highcharts-theme-salt .highcharts-color-7 {
+  fill: var(--salt-palette-categorical-2);
+  stroke: var(--salt-palette-categorical-2);
+}
+
+.highcharts-theme-salt .highcharts-color-8 {
+  fill: var(--salt-palette-categorical-20);
+  stroke: var(--salt-palette-categorical-20);
+}
+
+.highcharts-theme-salt .highcharts-color-9 {
+  fill: var(--salt-palette-categorical-6);
+  stroke: var(--salt-palette-categorical-6);
+}
+
+.highcharts-theme-salt .highcharts-point-inactive {
+  opacity: 1;
+  stroke: var(--salt-selectable-borderColor-disabled);
+}
+
+.highcharts-theme-salt .highcharts-container {
+  font-family: var(--salt-text-fontFamily);
+}
+
+.highcharts-theme-salt .highcharts-grid-line {
+  stroke: var(--salt-separable-tertiary-borderColor);
+}
+
+.highcharts-theme-salt .highcharts-title {
+  fill: var(--salt-content-primary-foreground);
+  font-size: var(--salt-text-h4-fontSize);
+  font-weight: var(--salt-text-h4-fontWeight);
```

We should set a line-height


```suggestion
  font-weight: var(--salt-text-h4-fontWeight);
  line-height: var(--salt-text-h4-lineHeight);
```